README.md

```diff
@@ -0,0 +1,30 @@
+# mender-stress-test-client
+
+[![Build Status](https://gitlab.com/Northern.tech/Mender/mender-stress-test-client/badges/master/pipeline.svg)](https://gitlab.com/Northern.tech/Mender/mender-stress-test-client/pipelines)
+
+The *mender-stress-test-client* is a dummy-client intended for simple testing of
+the Mender server functionality, or scale testing. It is not in any way a fully
+featured client. All it does is mimic a client's API, and download the update to
+/dev/null. This means that it has **no state** whatsoever, all updates are
+instantly thrown away, and the client will forever function as it did once
+booted up. This means that all parameters provided to the client at startup will
+stay this way until the client is brought down.
+
+## Getting Started
+
+### Building
+
+The `mender-stress-test-client` can be built by simply running `go build .` in
+the mender-stress-test-client repository. This will put an executable binary in
+the current repo. If a more universal option is wanted, the binary can be
+automatically install into the `~/$GOPATH/bin` repository by running `go install
+.`, and if this is already in your `$PATH` variable, then the stress-test-client
+will function exactly like any other binary in your `PATH`.
+
+### Running
+
+If the client is already installed, run it with the default options like so:
+`mender-stress-test-client --count=<device-count>`
```

Comment:
Actually, all this file is using `mender-` prefix, while not in the project.

To be honest, I think there is no point in maintaining the two stress clients. The purpose of `mender-stress-test-client` was scale testing and gui (manual) testing. So I would vote to completely replace the codebase there with this more efficient client, which is the one we are going to use from now on in Load Testing and potentially covers the same needs for gui (manual) testing.